deployment/aws/README.md

```diff
@@ -0,0 +1 @@
+Work In Progress
```

Comment:
@wrijeff I am going to let you fill this. Changed the structure, `deployment/aws/ec2`